src/config.rs

```diff
@@ -149,19 +157,21 @@ impl StarshipConfig {
     fn config_from_file() -> Option<toml::Table> {
         let file_path = get_config_path()?;
 
-        let toml_content = match utils::read_file(file_path) {
+        let toml_content = match utils::read_file(&file_path) {
             Ok(content) => {
                 log::trace!("Config file content: \"\n{}\"", &content);
                 Some(content)
             }
             Err(e) => {
-                let level = if e.kind() == ErrorKind::NotFound {
-                    log::Level::Debug
+                if e.kind() == ErrorKind::NotFound {
```

Comment:
I think this is inconsistent with prompt behaviour
When an invalid environment variable is specified, starship currently uses the default config (even if `~/.config/starship.config` is specified it seems)
If we want an error like this, I think it should be in the prompt as well, I feel like the user would expect `print-config` to print the configuration that is being used by the prompt?
I also have a PR on this issue, my bad: https://github.com/starship/starship/pull/5001